### ISSUE TITLE
Default materials under 1.13

### ISSUE THREAD

**Issue by x1p** [Trophonix/TradePlus#26]
The default config under 1.13 has a minor error.
```
[10:55:26 WARN]: [TradePlus] Unknown material [EXP_BOTTLE]. Make sure you've updated to the new 1.13 standard. Numerical item IDs are no longer supported. Using fallback: PAPER
[10:55:26 WARN]: [TradePlus] Unknown material [EXP_BOTTLE]. Make sure you've updated to the new 1.13 standard. Numerical item IDs are no longer supported. Using fallback: PAPER
```
EXP_BOTTLE must be EXPERIENCE_BOTTLE under 1.13

**Comment by Trophonix** [comment]
Ahh yeah, I can make it check the version for that. I set it to that
because I added a system that automatically updates material names but its
not fully working at the moment, it will be fixed soon hopefully when the
library Im using updates the fix onto their maven

On Nov 29, 2018 4:00 AM, "X1p" <notifications@github.com> wrote:

The default config under 1.13 has a minor error.

[10:55:26 WARN]: [TradePlus] Unknown material [EXP_BOTTLE]. Make sure
you've updated to the new 1.13 standard. Numerical item IDs are no
longer supported. Using fallback: PAPER
[10:55:26 WARN]: [TradePlus] Unknown material [EXP_BOTTLE]. Make sure
you've updated to the new 1.13 standard. Numerical item IDs are no
longer supported. Using fallback: PAPER

EXP_BOTTLE must be EXPERIENCE_BOTTLE under 1.13

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/Trophonix/TradePlus/issues/26>, or mute the thread
